Projects/RegExp-Helper-App.md

```diff
@@ -0,0 +1,40 @@
+# Regular Expression Helper
+
+**Tier:** 2-Intermediate
+
+Regular Expressions should be a valuable part of any developers toolbox. They
+provide a concise way to describe a pattern that can be used by JavaScripts
```

Comment:
It should be language agnostic. Maybe people would do this in C#.

Reply:
Great catch! Change implemented